runner.sh

```diff
@@ -7,4 +7,6 @@ cd "$(dirname ${0})/"
 conda deactivate
 conda activate illixr-runner
 
+export ILLIXR_INTEGRATION=yes
```

Comment:
Let's put this in the `build_one_plugin`. If someone wants to know how to "build one plugin," they will look there, so it is more discoverable there.